pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```diff
@@ -188,6 +193,8 @@ spec:
         - name: liveness-prometheus
           securityContext:
             privileged: true
+            capabilities:
+              drop: ["ALL"]
```

Comment:
there are other places also toolbox container and other places, did you checked we don't need there?

Reply:
added for the toolbox and for other places think we are covered since we take from scc.go file